.github/scripts/ubuntu-22.04/setup_runtime.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Install required packages for CodeCompass runtime
+sudo apt-get install -y git cmake make g++ graphviz \
+  libboost-filesystem1.71.0 libboost-log1.71.0 libboost-program-options1.71.0 \
+  libllvm11 clang-11 libclang1-11 libthrift-0.16.0 default-jre libssl1.1 libmagic1 \
+  libgit2-28 ctags googletest libldap-2.4-2
```

Comment:
Most of these package versions does not exist on Ubuntu 22.04, they should be updated.

CI only passes, because for some reason the `parse` job calls the `setup_build.sh`, instead of the `setup_runtime.sh`. Maybe because we need to build CodeCompass in the parse job, as that is one of the tests.

In that case if `setup_runtime.sh` is not needed, it could be deleted?

Reply:
I've totally overlooked on this. It is indeed an unnecessary script in this way. Removed.